Applications/Tizen_native/CustomShortcut/README.md

```diff
@@ -16,24 +18,28 @@ If you want to do in CLI, you first need to [convert the project to CLI](https:/
 
 for the time being, you have to install and prepare nntrainer, capi-nntrainer to run this program
 
+### Prerequisite
+
+- [NNTrainer Prerequisite](https://github.com/nnstreamer/nntrainer#prerequisites)
+- tizen studio 3.7
+- wearable 6.0 rootstrap (built after Aug 25 2020)
+- appropriate wearable tizen device with tizen 6.0 installed (built after Aug 25 2020)
+
+### Install Newest nntrainer to the device
+
 ```bash
 $ gbs build --arch armv7l
 $ sdb devices
 List of devices attached
 #device list
 $ sdb root on
-$ pushd ${gbs repo directory}
-$ sdb push nntrainer.0.0.1-0.armv7l.rpm /usr
-$ sdb push capi-nntrainer.0.0.1-0.armv7l.rpm /usr
-$ popd
-$ sdb shell
-sh-3.2# mount -o remount,rw /
-sh-3.2# cd /usr
-sh-3.2# rpm -e nntrainer
-sh-3.2# rpm -i nntrainer.0.0.1.-0.armv7l.rpm
-sh-3.2# rpm -i capi-nntrainer.0.0.1.-0.armv7l.rpm
+$ sdb shell "mount -o remount,rw /"
+$ sdb push ${gbs repo directory} /tmp_repos/
```

Comment:
I think now we dont need this as nntrainer comes with wearable image.
For some reason, capi-nntrainer is not part of the image.
@again4you can you give some hints why this is happening?

Reply:
You are right. I'll package capi and then update this line